framework/py/flwr/cli/run/run.py

```diff
@@ -178,23 +213,34 @@ def _run_with_control_api(
                 f"🎊 Successfully started run {res.run_id}", fg=typer.colors.GREEN
             )
         else:
-            typer.secho("❌ Failed to start run", fg=typer.colors.RED)
+            if remote_app_ref:
+                typer.secho(
+                    "❌ Failed to start run. Please check that the provided "
+                    "app identifier (@user_name/app_name) is correct.",
+                    fg=typer.colors.RED,
+                )
+            else:
+                typer.secho("❌ Failed to start run", fg=typer.colors.RED)
             raise typer.Exit(code=1)
 
         if output_format == CliOutputFormat.JSON:
-            run_output = json.dumps(
-                {
-                    "success": res.HasField("run_id"),
-                    "run-id": res.run_id if res.HasField("run_id") else None,
-                    "fab-id": fab_id,
-                    "fab-name": fab_id.rsplit("/", maxsplit=1)[-1],
-                    "fab-version": fab_version,
-                    "fab-hash": fab_hash[:8],
-                    "fab-filename": get_fab_filename(config, fab_hash),
-                }
-            )
+            # Only include FAB metadata if we actually built a local FAB
+            payload: dict[str, Any] = {
+                "success": res.HasField("run_id"),
+                "run-id": res.run_id if res.HasField("run_id") else None,
+            }
+            if not remote_app_ref and fab_id and fab_version and fab_hash:
```

Comment:
```suggestion
            if not remote_app_ref:
```

Would this be sufficient?